### PR TITLE
updated translations

### PR DESCRIPTION
**Pull request by pmgrove**
:koala:

Translation updates.

/cc @zendesk/quokka, @ydrain 

### References
 - Jira link: n/a

### Risks
 - None

### CODE REVIEW

**Comment by adammw** [comment]
:+1:

File: translations/hu.json [hunk 1]
```diff
@@ -84,6 +84,7 @@
     }
   },
   "delimiter": "Eredeti leírás",
+  "forbidden": "You are not authorized to view this linked ticket.",
```
Comment (sandlerr) [inline translations/hu.json]: ?

Comment (sandlerr) [inline translations/hu.json]: I never used to be able to read Hungarian

**Comment by sandlerr** [comment]
:+1:

File: translations/hu.json (hunk 1 above)
Comment (ydrain) [inline translations/hu.json]: Haha...Hungarian is done by a crowdsourced translator, and they are slow translating all the strings. Please ignore.

**Pull request merged**